src/TabCollectionView.js

```diff
@@ -8,8 +8,6 @@ define(function (require) {
 
         tagName: 'ul',
 
-        className: 'tab-item-view',
```

Comment:
I don't mind doing this, sorry. Fixed it, now `tabContainerCls` has default value as well